### ISSUE TITLE
Strange issue on iPhone

### ISSUE THREAD

**Issue by iandundas** [kean/ThreeColumnNavigation#1]
Just to note the strange behaviour on iPhone (though it works perfectly on the iPad)

![CleanShot 2021-02-25 at 13 35 55](https://user-images.githubusercontent.com/1131967/109154387-82d46e80-776e-11eb-9660-062c1fefb10c.gif)

odd!

**Comment by kean** [comment]
Yes, this is not meant to work on iPhone. I'm surprised it works at all. Thanks for posting it so that there is now a reference.
